rabbus.go

```diff
@@ -71,6 +72,11 @@ type (
 		PassiveExchange bool
 		// Queue the queue name
 		Queue string
+		// DeclareArgs is a list of arguments accepted for when declaring the queue.
+		// See https://www.rabbitmq.com/queues.html#optional-arguments for more info.
+		DeclareArgs amqp.Table
+		// BindArgs is a list of arguments accepted for when binding the exchange to the queue
+		BindArgs amqp.Table
```

Comment:
I didn't find any link for docs around which args are accepted here

Reply:
This is just free-form map, all the values are in the RabbitMQ docs. I think we can add a builder for common cases, like queue message TTL, but in another PR.

Reply:
Please, note this is the `BindArgs`. The `TTL` goes with the `DeclareArgs` right?

But sure, it's a free-map, just wanted to find some docs with examples of args that could be passed here, as for the `DeclareArgs`.